<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureColorClass</class>
 <widget class="QWidget" name="CaptureColorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaptureColor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="3">
      <widget class="ColorWidget" name="widgetColor" native="true">
       <property name="minimumSize">
        <size>
         <width>49</width>
         <height>49</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>49</width>
         <height>49</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3">
      <widget class="ColorWidgetPoint" name="widgetColorPoint" native="true">
       <property name="minimumSize">
        <size>
         <width>49</width>
         <height>49</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>49</width>
         <height>49</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mouse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_ms">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>RGB</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_rgb">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>HSV</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_hsv">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorWidgetPoint</class>
   <extends>QWidget</extends>
   <header>colorwidgetpoint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="capturecolor.qrc"/>
 </resources>
 <connections/>
</ui>
